schemas/pe.yml

```diff
@@ -39,9 +36,26 @@
       description: Internal product name of the file, provided at compile-time.
       example: Microsoft® Windows® Operating System
 
-
     - name: company
       level: extended
       type: keyword
       description: Internal company name of the file, provided at compile-time.
       example: Microsoft Corporation
+
+    - name: imphash
+      level: extended
+      type: keyword
+      short: A hash of the imports in a PE file.
+      description: >
+        A hash of the imports in a PE file. An imphash is a sort of fuzzy hash that can
+        be used to fingerprint binaries even after recompilation or other code-level
+        transformations have occurred that might change more traditional hash values.
+
+        Learn more at https://www.fireeye.com/blog/threat-research/2014/01/tracking-malware-import-hashing.html.
+      example: 0c6803c4e922103c4dca5963aad36ddf
+
+    - name: architecture
+      level: extended
+      type: keyword
+      description: CPU architecture target for the file.
```

Comment:
Is it worth adding a note that this is not necessarily the architecture of the machine itself?

Reply:
I'll leave this one up to you guys.

From the schema POV, there's sometimes a reflex to over-explain, as if we were telling people how to implement the source itself (e.g. a compiler actually populating PE headers in an executable), when in fact the schema's role is simply to explain where to get the data (e.g. getting the "architecture" header from the PE headers) and how to interpret it when looking at events that populate these.

But if you think there's a disconnect to explain or point out between `pe.architecture` and `host.architecture`, for example, yeah I think this may make sense.

Reply:
@rw-access I think that the fact that it says, 'for the file' seems fine to me, and I don't think we should over-explain as I would imagine people who were filling in this field via parsing pe headers would know how to do it. But, if you're thinking it's still vague, we can tighten it up.